pyproject.toml

```diff
@@ -18,7 +18,13 @@ requests = "^2.0.0"
 # structlog uses calendar versioning,
 # so using ^ is unnecessarily restrictive
 structlog = ">= 25.1.0"
-redis = "^4.2.0"
+# 6.0.0 and above only officially supports Redis 7.2, 7.4, and 8.0.
```

Comment:
We could update to 6.0.0 anyway, if we wanted to take the risk of using it with an unsupported Redis server version, but we need to more carefully consider that risk.

There isn't a compelling need to update. We aren't using any new features in 6.0.0, and what we currently have works. We could be forced to update if there was a security problem, but I'm not seeing any in e.g. https://www.cvedetails.com/version-list/25596/137563/1/Redis-Redis-py.html so far.

My current inclination is to stay on 5.x for now. And if future problems are discovered, I'd also consider looking into non-Redis solutions.